components/RichText/RichText.module.scss

```diff
@@ -22,6 +22,10 @@
     :global .themeDark :local & {
         color: var(--color--white);
     }
+
+    :global .themeDark .themeLight :local & {
+        color: var(--color--grey);
+    }
```

Comment:
`.themeLight` will be overruled by `.themeDark` if `.themeDark` is a parent of `.themeLight`. Although `.themeLight` is the closer parent class to the element to be styled, we need to be more specific to have the rule trigger as expected - hence the check here for the `.themeDark .themeLight`. It would be great if there was another trick to get specificity to work with the nearest parent, as this double theming (although rare) isn't the nicest way of duplicating style content.